Jenkinsfile

```diff
@@ -7,15 +7,17 @@ catchError {
     def images = ["docker-base-runtime", "docker-base-build",
                   "docker-base-gpu-build", "docker-base-gpu-runtime"]
     for (image in images) {
+        // Don't publish images containing CUDA
```

Comment:
Presumably this is a licensing issue ? Are there any mitigations you can think of ?